.github/workflows/pull_request.yaml

```diff
@@ -6,46 +6,32 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      # For PRs, to save GHA resources, only run on 1 python version
-      # Also, have the pip modes focus on as many different python
-      # versions as possible, package mode can be just 1 version.
       matrix:
         include:
-          - env:
-              TEST_TYPE: release-static
-            python: "2.7"
-          - env:
-              TEST_TYPE: source-static
-            python: "2.7"
-          - env:
-              TEST_TYPE: release-dynamic
-            python: "3.6"
-          - env:
-              TEST_TYPE: source-dynamic
-            python: "3.6"
-          - env:
-              TEST_TYPE: packages-static
-            python: "3.7"
-          - env:
-              TEST_TYPE: packages-dynamic
-            python: "3.7"
-          - env:
-              TEST_TYPE: packages-upgrade
-            python: "3.7"
-          - env:
-              TEST_TYPE: release-upgrade
-            python: "3.8"
-          - env:
-              TEST_TYPE: source-upgrade
-            python: "3.8"
+          - test_type: release-static
+            ansible: "2.8"
+          - test_type: source-static
+            ansible: "2.8"
+          - test_type: packages-static
+            ansible: "2.9"
+          - test_type: release-dynamic
+            ansible: "2.9"
+          - test_type: source-dynamic
+            ansible: "2.8"
```

Comment:
does these ansible versions cover python 2 and 3?

Reply:
Sort answer: No.
As i tried to express, we never really covered different python versions. Yes, we ran the `ansible-playbook` command with four different versions of python, but in the containers, ansible always used whatever python it found first / liked more.
Also we do not have the slightest snippet of python code in this collection, right? So all the code we execute has been tested upstream to work with whatever python version ansible is shipped for. We really should not care.

Reply:
you are right, thank you!